### PR TITLE
improve chainID explanation

### PR DESCRIPTION
**Pull request by jesusholoo**
improve chainID explanation for TOB audit!

### CODE REVIEW

File: docs/developers/protocol/hermez-protocol/protocol.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ Below is a summary of each transaction field and its explaination:
 ```
 signature_constant = sha256("I authorize this hermez rollup transaction")[:32/8]
 ```
-- `chainId`: set by the contract in order to prevent replay attacks in case of hardforks (16 bits)
+- `chainId`: ethereum chain identifier in order to prevent replay attacks in case of hardforks, we use only 2 bytes because according to EIP-155, v is used to set the CHAIN_ID according to this formula: `v = CHAIN_ID * 2 + 35|36` . Since `v` is 1 byte, there are only 110 safe different options of `CHAIN_ID`. Any network that has a value higher than this one will be overflowing v, so it’ll be susceptible to replay attacks (16 bits)
```
Comment (jleebee) [inline docs/developers/protocol/hermez-protocol/protocol.md]: "Ethereum"

**Review by jleebee — CHANGES_REQUESTED** [review]
Just add a quick capital E to "ethereum", the rest looks great!

**Comment by krlosMata** [comment]
> Just add a quick capital E to "ethereum", the rest looks great!

fixed!

**Review by krlosMata — DISMISSED** [review]
:smile_cat:

**Review by eduadiez — APPROVED** [review]
LGTM

**Review by jleebee — APPROVED** [review]

**Pull request merged**